Asgardia.Constitution.Proposal.txt

```diff
@@ -342,18 +343,18 @@ Article 31. Legal System and Branch structures
+§ 5. The second and subsequent Heads of State may be any Asgardian Citizen that submits his/her candidacy and qualifications to the Parliament for consideration. The Parliament shall appoint a committee of no more than seven (7) members to confirm the qualifications and, upon committee approval, provide the recommendation of the Candidates to the Council of Supreme Values. The Council will further interview the Candidates and confirm their fitness to lead the honourable office as defined by the Supreme Values of Asgardia. Three Earth-months prior to the conclusion of the current Head of State term, the Council of Supreme Values shall present the Candidates to the public for a final vote. There will be a maximum of five (5) candidates including the current Head of State, if he/she has not already served a second term. Two Earth-months prior to the conclusion of the current Head of State, the general election shall commence. Citizen votes shall be collected over a period of 48 Earth-hours and the candidate with the most votes shall be the next Head of State. In the even that two or more Candidates have the same number of votes, the Chairperson of the Council of the Supreme Values shall cast the tie-breaker vote.
 § 6. Upon retirement of the HoS, except in the case of Impeachment, he/she shall enjoy guaranteed personal safety and the protection of his/her private property for the remainder of his/her life.
 [Edit: Is that necessary in the constitution as it is a matter of law?]
 § 7. While in Office, the Head of State shall be referred only by titles of “Asgardian Head of State” or “Head of Asgardia.”
-§ 8. In the event the Head of State dies, resigns, becomes permanently incapacitated or is declared missing, as well as in the circumstances stipulated by law where the Head of State is unable to perform his/her duties, the duties of Head of State temporarily pass to the Chairperson of the Council of Supreme Values until the reasons which prevented the Head of State from performing such duties cease to be in effect or until the inauguration of the new Head of State. In the event of return from the above circumstances the Head of State must prove his/her fitness within two Earth-months and by the approval of a certified medical physician duly appointed by the Ministry of Science. In the event that both the Head of State and Chairperson of Council of Supreme Values are unable to perform such duties defined herein for the Head of State, the responsibilities shall fall to the following line of succession:

```

Comment:
For the comment in Section 6 above, I think I agree that this should be removed from the constitution, but I do see merit in this being a formal law in the future.